source/viz.md

```diff
@@ -848,37 +889,50 @@ glue('can_lang_plot_category', can_lang_plot_category, display=False)
 ```
 
 :::{glue:figure} can_lang_plot_category
-:figwidth: 700px 
+:figwidth: 700px
 :name: can_lang_plot_category
 
 Scatter plot of percentage of Canadians reporting a language as their mother tongue vs the primary language at home colored by language category.
 :::
 
 
 The legend in {numref}`can_lang_plot_category`
-takes up valuable plot area. 
+adds a fair bit of horizontal space.
```

Comment:
I'm not sure this story works well in the python version. If you look in the R version, the figure gets *super* squished. In the python version, honestly the side legend vs the top legend both look about the same to me.

We should fix this at some point, but if you don't want to handle it now you can open a separate issue for it and punt it for now.

Reply:
I will just reword it as an option